options/options.css

```diff
@@ -0,0 +1,173 @@
+:root {
+  --light-theme-profile-odd: hsl(0, 0%, 95%);
+  --light-theme-profile-border: hsl(0, 0%, 0%);
+  --light-theme-color: hsl(0, 0%, 0%);
+  --light-theme-background: hsl(0, 0%, 100%);
+  --light-theme-background-input: hsl(0, 0%, 100%);
+  --light-theme-normal-button: hsl(300, 67%, 42%);
+  --light-theme-delete-button: hsl(0, 100%, 50%);
+  --light-theme-button-color: hsl(0, 0%, 100%);
+  --light-theme-normal-button-hover: hsl(300, 67%, 22%);
+  --light-theme-delete-button-hover: hsl(0, 100%, 30%);
+  --light-theme-warning-background: hsl(62, 90%, 85%);
+  --light-theme-warning-text: hsl(49, 26%, 20%);
+  --light-theme-warning-url: hsl(240, 100%, 40%);
+  --light-theme-warning-url-visited: hsl(271, 68%, 32%);
+
+  --dark-theme-profile-odd: hsl(255, 6%, 8%);
+  --dark-theme-profile-border: hsl(0, 0%, 56%);
+  --dark-theme-color: hsl(0, 0%, 89%);
+  --dark-theme-background: hsl(255, 6%, 13%);
+  --dark-theme-background-input: hsl(255, 6%, 28%);
+  --dark-theme-normal-button: hsl(300, 67%, 22%);
+  --dark-theme-delete-button: hsl(0, 100%, 30%);
+  --dark-theme-button-color: hsl(0, 0%, 89%);
+  --dark-theme-normal-button-hover: hsl(300, 67%, 42%);
+  --dark-theme-delete-button-hover: hsl(0, 100%, 50%);
+  --dark-theme-warning-background: hsl(49, 26%, 20%);
+  --dark-theme-warning-text: hsl(45, 97%, 70%);
+  --dark-theme-warning-url: hsl(93, 65%, 50%);
+  --dark-theme-warning-url-visited: hsl(93, 45%, 30%);
+
+  /* Use light theme as default */
+  --profile-odd: var(--light-theme-profile-odd);
```

Comment:
@woodruffw I've changed the way the css custom properties were used. Now it should be easier to understand.

Reply:
Thanks, looks fine to me.